_posts/2020-02-15-pmemkv-bindings.md

```diff
@@ -0,0 +1,155 @@
+see [pmemkv-nodejs github repository][nodejs_gh].
+
+### pmemkv-python
+
+This is the second binding to be released in version 1.0. It has all functionalities
+from pmemkv 1.0.
+
+It also delivers API description in form of PyDoc documentation (accessible e.g. as html in
+[here](py_pmem_io)) and Python-native data type (`dict`) to store engine's configuration items.
+It handles returned statuses as exceptions and does not return any value when no errors occur.
+To install this project in your system it's required to type in: `python3 setup.py install [--user]`,
+and then it is installed in a system path or (when optional parameter `--user` is given) within
+user's files (`~/.local/lib/`).
+
+For more information, like required dependencies, usage and full installation guide,
+see [pmemkv-python github repository][py_gh].
+
+### pmemkv-java
+
+Latest released version is 0.9 and it's working just fine with libpmemkv 1.0, but
+its API may still change. In this binding's code, the JNI framework is used,
+with implementation of pmemkv's C API delivered in separate repository.
+It handles returned statuses as exceptions and does not return any value when no errors occur.
+Installation process, when pmemkv and pmemkv-jni are installed in the system, comes down to
+typing, within source directory, command: `mvn install`.
+
+For more information, like required dependencies, usage and full installation guide,
+see [pmemkv-java github repository][java_gh]. To see the repository of JNI content,
+see [pmemkv-jni github repository][jni_gh].
+
+### pmemkv-ruby
+
+It's currently released as 0.9 version, properly working with pmemkv 1.0,
+but its API is still a work in progress and it may change.
+It handles returned statuses as exceptions and returns numeric value of `OK` status,
+when no errors occur.
+Installation of this project and its dependencies is done using bundler
+(`gem install bundler -v '< 2.0'`) and it comes down to typing: `bundle install`.
+
+For more information, like required dependencies, usage and full installation guide,
+see [pmemkv-ruby github repository][ruby_gh].
+
+### Summary
+
+| Language  | Version | URL |
+| --------- | ------- | --- |
+| **Node.js** | **1.0** | **[GitHub][nodejs_gh]**, **[docs][nodejs_pmem_io]** |
+| **Python** | **1.0** | **[GitHub][py_gh]**, **[docs][py_pmem_io]** |
+| Java | 0.9 | [GitHub][java_gh] |
+| Ruby | 0.9 | [GitHub][ruby_gh] |
+
+### Looking forward
+
+In future more bindings may come to life. There are currently two (of which we are aware)
+still in development and outside of our organization, you can read about them here:
+* [Rust binding][rust_binding_iss]
+* [Go binding][go_binding_iss]
+
+If you feel like we're missing some language, that may be useful to have in our portfolio,
+or you just want to let us know we don't have information about some (possibly yours)
+pmemkv's binding, or your feel like we're missing some functionalities, don't hesitate to
```

Comment:
`please file an issue to let us know. We are always eager to help our community grow.` or something like that.